implementations/rust/ockam/ockam_node/src/storage/database/sqlx_database.rs

```diff
@@ -209,15 +258,64 @@ impl SqlxDatabase {
             .map_err(Self::map_sql_err)?
             .log_statements(LevelFilter::Trace)
             .log_slow_statements(LevelFilter::Trace, Duration::from_secs(1));
-        let pool = Pool::connect_with(options)
+
+        let max_pool_size = match configuration {
+            DatabaseConfiguration::Sqlite {
+                single_connection, ..
+            } => {
+                if *single_connection {
+                    1
+                } else {
+                    16
```

Comment:
Could you please add a comment explaining why 16 is picked here?

Reply:
Maybe an env var with a default value could be useful in environments where we want to customize it?

Reply:
> Maybe an env var with a default value could be useful in environments where we want to customize it?
We could, but I don't think there is the need for fine-tuning yet.
The only reason i changed from the default 10 to 16, it's because it's closer to the real number of threads tokio is using on a typical node.